pyproject.toml

```diff
@@ -42,13 +42,12 @@ dependencies = [
     "aiohttp",  # TODO: remove in favor of httpx
     "anyio",
     "coredis",
-    "fhaviary[llm]>=0.8.2",  # For namespace reorg
+    "fhaviary[llm]>=0.10.0",  # For invalid tool fixes
     "html2text",  # TODO: evaluate moving to an opt-in dependency
     "httpx",
     "limits",
     "litellm>=1.44",  # to prevent sys.stdout on router creation
     "numpy",
-    "openai<1.47",  # TODO: remove after https://github.com/BerriAI/litellm/issues/5854 resolution
```

Comment:
@whitead can you remove this from `pre-commit`'s `mypy` as well?

Reply:
Isn't in pre-commit

Reply:
Are you sure